src/components/EventInfo/Image.jsx

```diff
@@ -1,8 +1,7 @@
 import styled from "styled-components";
 
 const Image = styled.img`
-  max-width: 720;
```

Comment:
Would recommend we keep this, as we have kept the max-width either way, and the width is duplicated in the JSX file.

Reply:
Width in JSX file is for allotting space so user doesn't get flash of content when browser lazy loads image as image would change height before and after load (see https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/)